tests/store/test_store_api.py

```diff
@@ -48,7 +48,7 @@ def test_fetch_no_files_before_oldest(populated_store, bundle_data_old, old_time
     # WHEN fetching all files before the oldest date
     date = old_timestamp - datetime.timedelta(days=10)
     assert date < old_timestamp < timestamp
-    query = store.files_before(before=str(date))
+    files = store.files_before(before_date=date)
```

Comment:
same as above. I also think files_before should be renamed to `get_files_before`

Reply:
Added hints and updated name.